cfme/tests/services/test_service_customer_bz.py

```diff
@@ -67,3 +71,46 @@ def test_edit_bundle_entry_point(appliance, provider, catalog_item, request):
                                                               "ServiceProvision_Template/"
                                                               "CatalogItemInitialization")
     view.cancel_button.click()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1705021)])
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1705021.zip", "bz_1705021", None)],
+    ids=["sample_domain"],
+)
+@pytest.mark.parametrize("file_name", ["bz_1705021.yml"],
+    ids=["sample_dialog"],
+)
+def test_refresh_dynamic_field(appliance, import_datastore, import_data,
+                               catalog_item_with_imported_dialog):
+    """Tests refresh dynamic field when field name has 'password' in label.
+    Metadata:
+        test_flag: provision
+
+    Bugzilla:
+        1705021
+
+    Polarion:
+        assignee: nansari
+        casecomponent: Services
+        initialEstimate: 1/4h
+        tags: service
+    testSteps:
+        1. Import bz dialog
+        2. Import  bz datastore
+        3. Create catalog item
```

Comment:
All of these steps are actually happening in setup, in the fixtures. Looks like the only thing happening in Call phase here is ordering.